packages/custom-functions-metadata/test/cases/requiresstreamaddress/expected.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Please call this requiresaddress-streaming (which is converted from error-requiresaddress-streaming), or consider merging both non-streaming and streaming cases into single requiresaddress test case.